src/reducers/index.js

```diff
@@ -4,11 +4,15 @@ import authReducer from './auth';
 import signupReducers from './signupReducers';
 import articleReducer from './articleReducer';
 import profileReducer from './profile';
+import passwordReset from './passwordReset';
+import updatePasswordReducer from './updatepassword';
 
 export default combineReducers({
   auth: authReducer,
   form: formReducer,
   signup: signupReducers,
   article: articleReducer,
   profile: profileReducer,
+  reset: passwordReset,
```

Comment:
Is there really a need to have two different states on our store just for handling the password reset functionality?

How about merging both into one, or even better, making use of the `auth` state?

What do you think?

Reply:
@okoroemeka Like @akhilome said I'll go with you merging both into auth.

Reply:
resolved.

> Hey @okoroemeka, I left a few comments on your PR please attend to them. Also, I noticed when I don't have an account on **AH** and try to reset my password, I don't get any error. I should get a feedback of some sort regardless of whether I'm an existing user or not.
> 
> See below screenshot
> 
> <img alt="screen shot 2019-01-30 at 3 31 01 am" width="754" src="https://user-images.githubusercontent.com/5154605/51954863-1cbb7700-2442-11e9-908d-65e023eedbd7.png">

@codeshifu I am displaying the same message for an existing user and a non-existing. And this is for security reasons. I actually visited many sites to understand how the password reset stuff works and I realised they are doing the same thing.